include/alpaka/dev/DevHipRt.hpp

```diff
@@ -13,6 +14,23 @@ namespace alpaka
 {
     //! The HIP RT device handle.
     using DevHipRt = DevUniformCudaHipRt<ApiHipRt>;
+
+    namespace trait
+    {
+        //! The HIP RT preferred device warp size get trait specialization.
+        template<>
+        struct GetPreferredWarpSize<DevUniformCudaHipRt<ApiHipRt>>
+        {
+            ALPAKA_FN_HOST static auto getPreferredWarpSize(DevUniformCudaHipRt<ApiHipRt> const& dev) -> std::size_t
+            {
+                int warpSize{0};
+                ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK_TAPI(
+                    ApiHipRt,
+                    ApiHipRt::deviceGetAttribute(&warpSize, ApiHipRt::deviceAttributeWarpSize, dev.getNativeHandle()));
```

Comment:
instead of adding `ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK_TAPI` you could do
```suggestion
                using TApi = ApiHipRt;
                ALPAKA_UNIFORM_CUDA_HIP_RT_CHECK(
                    ApiHipRt::deviceGetAttribute(&warpSize, ApiHipRt::deviceAttributeWarpSize, dev.getNativeHandle()));
```

Reply:
Ok, thanks.